its/src/test/java/com/sonar/it/csharp/MetricsIncludeHeaderCommentTest.java

```diff
@@ -43,6 +43,10 @@
  * This is copy pasted from MetricsTest. It does not re-test everything, it just makes sure
  * that when 'ignoreHeaderComments' is set to False, it counts the header comments as well
  * and it does not modify the LOC metrics.
+ *
+ * <p>
+ * Note: this class runs the analysis once in {@link MetricsIncludeHeaderCommentTest#getRuleChain()}, before the tests.
```

Comment:
this is added automatically by IntelliJ when cleaning up, it's probably a javadoc convention